Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -645,9 +645,9 @@ script:
       secret: false
     - auto: PREDEFINED
       default: false
-      description: Whether to return only mailboxes which contain the email. Default
+      description: Whether to return the content of the messages as well. Default
         is "False".
-      name: show-only-mailboxes
+      name: return-msg-content
```

Comment:
Please revert this change.
The change should only come from the default value to `true`, and not by changing the arg itself.